app/assets/stylesheets/components/_banner.scss

```diff
@@ -48,7 +39,11 @@
 }
 
 .app-c-banner__desc {
-  @include core-24;
+  @include core-19;
   // Ensure the text has a line-length of around 60 characters
   max-width: 30em;
```

Comment:
Is this max-width figure still correct now there is no aside? It looks quite short considering the width of the component 
<img width="916" alt="screen shot 2017-12-08 at 14 59 56" src="https://user-images.githubusercontent.com/29889908/33771316-8517381a-dc28-11e7-98bc-583251e2636c.png">

Reply:
I think let's leave it for now: max character limit works well for reading so I'm less concerned that it looks odd than that it is less readable. Also we may be changing the banners again soon and I don't want to lose this bit of code :)